src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -249,6 +395,9 @@ def cli_cosmosdb_update(client,
         else:
             raise CLIError(
                 'backup-interval and backup-retention can only be set for accounts with periodic backup policy.')
+    elif backup_policy_type is not None and backup_policy_type.lower() == 'continuous':
+        if isinstance(existing.backup_policy, PeriodicModeBackupPolicy):
+            backup_policy = ContinuousModeBackupPolicy()
```

Comment:
Can we update the `backup_policy` directly without checking `isinstance(existing.backup_policy, PeriodicModeBackupPolicy)`? Maybe there will be other policy types and it's not `PeriodicModeBackupPolicy`, we should update anyway.

Reply:
Currently, we only allow transitioning from PeriodicModeBackupPolicy -> ContinuousMode.
In future, when we allow more, we ease the check but currently its required for safety

Reply:
That makes sense~